examples/powertools-examples-core-utilities/sam/pom.xml

```
@@ -4,7 +4,7 @@
 
     <groupId>software.amazon.lambda.examples</groupId>
     <version>2.0.0-SNAPSHOT</version>
-    <artifactId>powertools-examples-core-utilities</artifactId>
+    <artifactId>powertools-examples-core-sam</artifactId>
```

```suggestion
    <artifactId>powertools-examples-core-utilities-sam</artifactId>
```
probably? Not sure where this change has come from though